spec/support/factory/user_factory.ex

```diff
@@ -2,7 +2,7 @@ defmodule Squadster.Support.Factory.UserFactory do
   defmacro __using__(_opts) do
     quote do
       def user_factory do
-        uid = Faker.Util.format("%9d")
+        uid = Faker.Random.Elixir.random_between(100000000, 999999999) |> Integer.to_string
```

Comment:
Try to `100_000_000` and `999_999_999` or `1e+8` and `9,99e+8`
Or something from `Faker`, (`Faker.Code.isbn10` looks like something we need)

Reply:
I think, good catch is `Faker.Util.format("%9d", d: fn() -> Faker.Random.Elixir.random_between(1, 9) end)`

Reply:
`Faker.Code.isbn10` can start with zero :(
`Faker.Util.format("%9d", d: fn() -> Faker.Random.Elixir.random_between(1, 9) end)` looks very good, but will not contain zeroes at all
Fixed this with little crutch